### ISSUE TITLE
Can I use the synthetic data generation pipeline to create my dataset?

### ISSUE THREAD

**Issue by ShiRuiyao** [meidachen/STPLS3D#27]
I noticed that you are developing a synthetic data generation pipeline to create synthetic training data. So I wonder whether we can use that pipeline to create my dataset. If its answer is yes, when will it be published and how can we get its code?

**Comment by meidachen** [comment]
Yes, we are planning to release the pipeline, but we are still working on the licenses of the assets we have used in the pipeline and scripts that connect each component I don't really have a date now when it can be released. What is the task you are trying to do? and what kind of data you want to generate? Maybe I can help you to generate it if it is already built into the pipeline, or maybe the data already exists for the datasets we have created.
